machine/delta/delta_tn48m/u-boot/onie-plarform-env.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/include/configs/mvebu_armada-8k.h b/include/configs/mvebu_armada-8k.h
+index daac8ba..9ad9af2 100644
+--- a/include/configs/mvebu_armada-8k.h
++++ b/include/configs/mvebu_armada-8k.h
+@@ -33,4 +33,10 @@
+ #define ERRATA_FE_3124064
+ #endif
+ 
++#define CONFIG_PLATFORM_ENV    \
++       "oniefile=delta_tn48m-r0.itb\0"        \
++       "consoledev=ttyS0\0"    \
++       "onie_start=0x00400000\0"       \
++       "onie_sz.b=0x00C00000\0"
++
+ #endif /* _CONFIG_MVEBU_ARMADA_8K_H */
```

Comment:
The missing new line, also highlighted by GitHub, caused the build to fail in my environment. What distribution and editor do you use?

Merge/pull request https://github.com/opencomputeproject/onie/pull/950 fixes it.